packages/odyssey-react-mui/src/SearchField.tsx

```diff
@@ -76,21 +82,58 @@ const SearchField = forwardRef<HTMLInputElement, SearchFieldProps>(
       id: idOverride,
       isDisabled = false,
       label,
-      onChange,
+      onChange: onChangeProp,
       onFocus,
       onBlur,
+      onClear: onClearProp,
       placeholder,
-      value,
+      value: controlledValue,
     },
     ref
   ) => {
+    const [uncontrolledValue, setUncontrolledValue] = useState("");
+
+    const onChange = useCallback(
+      (e) => {
+        setUncontrolledValue(e.target.value);
```

Comment:
Change `target` to `currentTarget`
```suggestion
        setUncontrolledValue(e.currentTarget.value);
```